src/app/components/Link/index.tsx

```diff
@@ -73,9 +74,15 @@ type CustomTrimEndLinkLabelProps = {
   highlightedPart?: string
 }
 
-const CustomTrimEndLinkLabel: FC<CustomTrimEndLinkLabelProps> = ({ name, to, highlightedPart }) => (
-  <TrimEndLinkLabel label={name} to={to} trimStart={14} highlightedPart={highlightedPart} />
-)
+const CustomTrimEndLinkLabel: FC<CustomTrimEndLinkLabelProps> = ({ name, to, highlightedPart }) => {
+  const trimmedLabel = <HighlightedTrimmedText text={name} pattern={highlightedPart} fragmentLength={14} />
```

Comment:
nit: I don't see a reason why trimmedLabel should be a variable at all. Why not just replace the bellow in jsx?